### DISCUSSION TITLE
https://starter.obytes.com/getting-started/customize-app/

### DISCUSSION THREAD

**Discussion by giscus** [obytes/react-native-template-obytes#394] [Docs Comments]
# https://starter.obytes.com/getting-started/customize-app/

Customize your app by updating Expo configuration and app icon and splash screen.

https://starter.obytes.com/getting-started/customize-app/

**Comment by orkhannabiyev** [comment]
The link the environment variables guide https://starter.obytes.com/getting-started/customize-app/#:~:text=config%20file%20following-,the%20environment%20variables%20guide,-and%20import%20your is not available.

**Reply by yjose** [reply]
fixed
